inc/inventorycomputerlib.class.php

```diff
@@ -459,7 +459,8 @@ function updateComputer($a_computerinventory, $computers_id, $no_history, $setdy
                foreach ($a_computerinventory['harddrive'] as $key => $arrays) {
                   $arrayslower = array_map('strtolower', $arrays);
                   foreach ($db_harddrives as $keydb => $arraydb) {
-                     if ($arrayslower['serial'] == $arraydb['serial']) {
+                     if (!isset($arraydb['serial'])
```

Comment:
Is not condition inverted? I guess we would update item only if serial is defined; am I wrong?

Reply:
in fact no, but i'll do more readable